src/shared/bus-unit-util.c

```diff
@@ -297,7 +326,32 @@ int bus_append_unit_property_assignment(sd_bus_message *m, const char *assignmen
                               "KeyringMode"))
                 r = sd_bus_message_append(m, "v", "s", eq);
 
-        else if (STR_IN_SET(field, "AppArmorProfile", "SmackProcessLabel")) {
+        else if (streq(field, "StandardInputData")) {
+                _cleanup_free_ char *cleaned = NULL;
+                _cleanup_free_ void *decoded = NULL;
+                size_t sz;
+
+                cleaned = strdup(eq);
+                if (!cleaned)
+                        return log_oom();
+
+                delete_chars(cleaned, WHITESPACE);
+
+                r = unbase64mem(cleaned, (size_t) -1, &decoded, &sz);
```

Comment:
yes, but that'd be some non-trivial refactoring (because the code as it is calculates the destination index of the decoding from the source index, which doesn't work if whitespace is in the stream), which I didn't also want to cram into this PR too. It's already hard to digest with the assorted stuff it contains.

Other uses of unbase64mem also delete the whitespace first, hence this is also means reworking a lot of other callers. but yes, we should do that, but i figure in a later commit